### PR TITLE
Use `**/*` as default basePath instead of `*`

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

**Comment by IonicaBizau** [comment]
Thanks! However, What's the use-case? Can you please describe what will be the differences?

After this change, will be included everything from the directory?

**Comment by PeterDaveHello** [comment]
If there is another directory in the basePath, we'll need it, `**` can match the path including directories.

**Comment by IonicaBizau** [comment]
@PeterDaveHello Cool! :sparkles: Please add yourself in the `contributors` array, in `package.json`.

**Comment by PeterDaveHello** [comment]
Sure, done.

File: package.json [hunk 1]
```diff
@@ -18,6 +18,13 @@
     "library"
   ],
   "author": "Ionică Bizău <bizauionica@gmail.com>",
+  "contributors": [
+    {
+      "name": "Peter Dave Hello",
+      "email": "hsu <at> peterdavehello <dot> org",
```
Comment (IonicaBizau) [inline package.json:24]: This will be normalized on the next `npm init`.

**Comment by IonicaBizau** [comment]
Thanks! :ship: :fireworks:

**Comment by IonicaBizau** [comment]
@PeterDaveHello BTW, do you use this tool often? How do you feel using it? I'd be glad to improve it making the life easier. :smile:

**Comment by PeterDaveHello** [comment]
Personally I don't use it because I am very familiar with cdnjs's structure haha ... I sent this PR is because some others used it but I think the default config can be better :+1:

**Comment by IonicaBizau** [comment]
Haha, cool! The main reason I created it is because of the laggy `git` commands in `cdnjs`. :smile:

**Comment by PeterDaveHello** [comment]
BTW I think you can add it to https://github.com/cdnjs/cdnjs/wiki/Extensions%2C-Plugins%2C-Resources.

**Pull request merged**